src/python/tools/Report_InvalidUrls_FindHttpURLs.csv

```diff
@@ -0,0 +1,18 @@
+url	relativeFilepath
+http://schemas.microsoft.com/developer/msbuild/2003	.\src\CommonCpp.props
+http://schemas.microsoft.com/developer/msbuild/2003	.\src\NativeBridge\NativeBridge.vcxproj
+http://schemas.microsoft.com/developer/msbuild/2003	.\src\python\nimbusml.pyproj
+http://www.nvidia.com/Drivers	.\src\DotNetBridge\MessageValidator.cs
```

Comment:
>http://www.nvidia.com/Drivers [](start = 0, length = 29)

this one is valid

Reply:
I believe these were due to me running FindHttpURLs.py continuously, which resulted in these website to deny me access for perhaps possible fear of a DDoS attack.

Reply:
so why this url is still here? No its not bcs of that. debug and see the error code. 

---
In reply to: [337776732](https://github.com/microsoft/NimbusML/pull/346#discussion_r337776732) [](ancestors = 337776732)